service/src/main/java/tds/session/repositories/impl/SessionRepositoryImpl.java

```diff
@@ -40,7 +59,42 @@ public SessionRepositoryImpl(DataSource datasource) {
             sessionOptional = Optional.empty();
         }
 
-
         return sessionOptional;
     }
+
+    /**
+     * In order to preserve compatibility with the existing TDS system, this implementation executes an UPDATE against
+     * the existing record in the {@code session.session} table.  Once the new code has complete responsibility for
+     * managing a {@link Session}, this implementation will be updated to comply with the "immutability" database design
```

Comment:
This is a small quibble about wording.  I understand what you're saying here, but I think it would be better placed above the SQL (since this isn't really a JavaDoc) variable explaining why we're doing an update and not following mutability.  I'd remove the "Once the new code" line because that provides no context to external users, and I don't think anyone knows without a doubt if we'll actually change the management of session.